src/pygambit/game.pxi

```diff
@@ -1008,17 +1011,22 @@ class Game:
             If `node` is a `Node` from a different game, or `player` is a `Player` from a
             different game.
         """
-        resolved_node = cython.cast(Node, self._resolve_node(node, 'append_move'))
-        resolved_player = cython.cast(Player, self._resolve_player(player, 'append_move'))
-        if len(resolved_node.children) > 0:
-            raise UndefinedOperationError("append_move(): `node` must be a terminal node")
-        if len(actions) == 0:
-            raise UndefinedOperationError("append_move(): `actions` must be a nonempty list")
-        resolved_node.node.deref().AppendMove(resolved_player.player, len(actions))
-        for label, action in zip(actions, resolved_node.infoset.actions):
-            action.label = label
-
-    def append_infoset(self, node: typing.Union[Node, str],
+        L = []
```

Comment:
Python variables by convention are snake_case.  By extension of our existing convention this should be called something like `resolved_nodes`